taskcluster/ci/swift/kind.yml

```diff
@@ -17,9 +17,12 @@ kind-dependencies:
 tasks:
   build:
     attributes:
+      run-on-pr-type: full-ci
       resource-monitor: true
+      release-routes:
+        - index.project.application-services.v2.swift.{appservices_version}
```

Comment:
This is the same route from before, so no need to block this PR. Though I notice that there's no `release-type` in the route. This means if you have both staging and prod release types, the former could overwrite the latter. Maybe it's not a problem in this case, but would be good to double check this.

Reply:
Good point.  I don't think it'll be an immediate issue, but it's good to fix.  I made this issue for it: https://github.com/mozilla/application-services/issues/5545.

I think we can do this as a follow-up, hopefully next week.